toml.md

```diff
@@ -616,6 +633,13 @@ represent that entire day without any relation to an offset or timezone.
 ld1 = 1979-05-27
 ```
 
+Valid years run from `0001` to `9999`.
+
+```toml
+# ld_year0 = 0000-12-31  # INVALID
+ld_year1 = 0001-01-01    # VALID; this is the earliest valid local date.
+```
+
```

Comment:
Again, I think that can be cut.

Reply:
I don't agree, but I could be persuaded otherwise. But see my comments on #1000, which I will make here if they make sense. In short, I also agree that year zero may be invalid, but could be valid in some scenarios.

Reply:
I did go ahead and cut this example.